api/model.go

```diff
@@ -157,7 +157,8 @@ type UpdateReplicaCountInput struct {
 }
 
 type PVCreateInput struct {
-	PVName string `json:"pvName"`
+	PVName   string `json:"pvName"`
+	PVFsType string `json:"fsType"`
```

Comment:
Use `FSType` as the field name.  It needs to be consistent with the json name.

Reply:
fixed